compiler/syntax/src/res_core.ml

```diff
@@ -2694,34 +2672,37 @@ and parse_jsx p =
   Parser.leave_breadcrumb p Grammar.Jsx;
   let start_pos = p.Parser.start_pos in
   Parser.expect LessThan p;
-  let jsx_expr =
+  let jsx_expr, jsx_attrs =
     match p.Parser.token with
     | Lident _ | Uident _ ->
-      parse_jsx_opening_or_self_closing_element ~start_pos p
+      (parse_jsx_opening_or_self_closing_element ~start_pos p, [jsx_attr])
     | GreaterThan ->
       (* fragment: <> foo </> *)
-      parse_jsx_fragment p
-    | _ -> parse_jsx_name p
+      (parse_jsx_fragment start_pos p, [])
+    | _ -> (parse_jsx_name p, [])
   in
   Parser.eat_breadcrumb p;
-  {jsx_expr with pexp_attributes = [jsx_attr]}
+  {jsx_expr with pexp_attributes = jsx_attrs}
 
 (*
  * jsx-fragment ::=
  *  | <> </>
  *  | <> jsx-children </>
  *)
-and parse_jsx_fragment p =
+and parse_jsx_fragment start_pos p =
   let children_start_pos = p.Parser.start_pos in
   Parser.expect GreaterThan p;
```

Comment:
@shulhi I believe the comment capture isn't working here for:

```
let _ =
  <> // foobar

  {React.string("m")}
  {React.int(4)}
  </>
```

I got nothing for:
```
dune exec res_parser -- A.res -print comments
leading comments:                        

comments inside:

trailing comments:

```